### PR TITLE
Update the selection-api IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build BROKEN

Started: 2018-03-05 21:41:51
Finished: 2018-03-05 21:48:16


<h2>Failing Jobs</h2>
<ul>

<li>firefox:nightly</li>

<li>chrome:dev</li>

<li>safari:11.0</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25289)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/349439145)

File: interfaces/selection-api.idl [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
-  stringifier DOMString ();
-};
+// GENERATED CONTENT - DO NOT EDIT
+// Content of this file was automatically extracted from the Selection API spec.
+// See https://w3c.github.io/selection-api/
 
-partial interface Document {
-  Selection? getSelection();
-};
+        interface Selection {
```
Comment (foolip) [inline interfaces/selection-api.idl]: Can you update this with the whitespace stripping fixes?

**Review by foolip — APPROVED** [review]
LGTM % whitespace and checking that selection/interfaces.html isn't broken.

**Comment by lukebjerring** [comment]
@foolip - this is now clearly a no-op after the latest scrape, merge if you're happy.

**Comment by foolip** [comment]
I am very happy :)

**Pull request merged**